vue/test/components/ripe-configurator/ripe-configurator.test.js

```diff
@@ -0,0 +1,47 @@
+const assert = require("assert");
+const base = require("../../base");
+
+class MockErrorRipeApi {
+    bindConfigurator() {
+        return {
+            bind() {},
+            resize() {}
+        };
+    }
+
+    config() {
+        throw new Error("Error");
+    }
+}
+
+describe("RipeConfigurator", () => {
+    it("should instantiate the component", () => {
+        const component = base.getComponent("RipeConfigurator", {
+            props: {
+                brand: "dummy",
+                model: "cube",
+                version: 52,
+                size: 1000
+            }
+        });
+
+        const loader = component.find(".loader");
+
+        assert.strictEqual(loader.exists(), true);
+        assert.strictEqual(component.emitted("loading").length, 1);
+    });
+
+    it("should return error when instantiating the component", () => {
+        const component = base.getComponent("RipeConfigurator", {

```

Comment:
@BeeMargarida  use https://nodejs.org/api/assert.html#assert_assert_rejects_asyncfn_error_message in a different PR